.github/CHANGELOG.md

```diff
@@ -186,6 +186,31 @@
   engine = sf.RemoteEngine("X8", connection=connection)
   ```
 
+* The `sf.api.Result` class has been updated to support the Xanadu Cloud Client integration.
+  [(#651)](https://github.com/XanaduAI/strawberryfields/pull/651)
+
+  While `Result.samples` should return the same type and shape as before, a `Result.samples_dict`
+  property has been added, returning the samples as a dictionary with corresponding measured modes
+  as keys.
```

Comment:
Is it worth mentioning that `Result.samples_dict` existed before (but it was called `Result.all_samples`)?